app/models/pipeline_run.rb

```diff
@@ -264,6 +264,20 @@ def update_genera
     ")
   end
 
+  def update_is_phage
+    TaxonCount.connection.execute("
+      UPDATE taxon_counts
+      INNER JOIN taxon_lineages ON taxon_counts.tax_id = taxon_lineages.taxid
+      SET taxon_counts.is_phage = IF(taxon_lineages.family_name IN ('Myoviridae', 'Siphoviridae', 'Podoviridae', 'Lipothrixviridae',
```

Comment:
I think these names should be extracted out as an array constants.

Reply:
Also, by looking at update_genera function, I think you can actually put family_taxid into taxon_counts in the same query and just update the query in the report_helper, it's cleaner that way.

Reply:
Also, family_taxid is in general pretty useful that we can put into each row in the report soon.

Reply:
Ok, set array constant and added family_taxid to taxon_counts. Do we still want to explicitly set the column is_phage? I think yes, so that we don't need to check the entire list of phage families on the fly each time we display a report?